less/navigation.less

```
@@ -1 +1,40 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+.content-nav {
+    background-color: cyan;
```

I think the stretch asked for a transparent fixed nav. Just thought I'd throw that in there.

```suggestion
   background-color: rgba(78, 255, 255, .7);
```